std_msgs/Header header
geometry_msgs/Vector3 force
geometry_msgs/Quaternion orientation
float64[3] k_r
float64[3] k_om
quadrotor_msgs/AuxCommand aux
